_data/communication.yml

```
@@ -4,6 +4,7 @@ websites:
     img: actionnetwork.png
     tfa:
       - totp
```

```suggestion
      - proprietary
```
The sentence
> Turn off multi-device in the Authy app for the highest level of security.

suggests to me that it's not actually `totp`, but rather `proprietary`, correct? I can't think of a way for this to work with regular `totp`.

---

My understanding of [Authy multi-device](https://authy.com/features/multiple-devices/) was that regular TOTP tokens would be stored in the cloud and could then be accessed from anywhere you were logged in, similar to how [1Password supports TOTP](https://support.1password.com/one-time-passwords/). Presumably for this site, you could use another TOTP authenticator, although it is recommending Authy and giving advice on securing Authy account? That's what I understood from it anyways.

---

Okay, let's leave it at is is then.